documentation/asciidoc/computers/config_txt/common.adoc

```diff
@@ -0,0 +1,51 @@
+== Common Options
+
+=== Common Display Options
+
+==== `disable_overscan`
+
+The default value for `disable_overscan` is `0` which gives default values of overscan for the left, right, top, and bottom edges of `48` for HD CEA modes, `32` for SD CEA modes, and `0` for DMT modes. 
+
+Set `disable_overscan` to `1` to disable the default values of xref:configuration.adoc#underscan[overscan] that are set by the firmware. 
+
+==== `hdmi_enable_4kp60` (Raspberry Pi 4 Only)
+
+By default, when connected to a 4K monitor, the Raspberry Pi 4B, 400 and CM4 will select a 30Hz refresh rate. Use this option to allow selection of 60Hz refresh rates. 
+
+IMPORTANT: It is not possible to output 4Kp60 on both micro HDMI ports simultaneously.
+
+WARNING: Setting `hdmi_enable_4kp60` will increase power consumption and the temperature of your Raspberry Pi. 
+
+=== Common Hardware Configuration Options
+
+==== `auto_detect_cameras`
```

Comment:
The default `config.txt` on _bullseye_ includes `camera_auto_detect=1` (with the same explanation), but here is listed as `auto_detect_cameras`.
Typo or both options are valid ?

Reply:
Typo - they should be `camera_auto_detect` and `display_auto_detect`.

Reply:
Fixed up.